common/views/components/ToolbarSegmentedControl/ToolbarSegmentedControl.tsx

```diff
@@ -5,10 +5,13 @@ import { font } from '@weco/common/utils/classnames';
 import Space from '../styled/Space';
 import { IconSvg } from '@weco/common/icons';
 
-const List = styled.ul.attrs({
-  className:
-    'no-margin no-padding plain-list flex-inline flex--v-center flex--h-center',
-})`
+const List = styled.ul`
+  margin: 0 !important;
```

Comment:
Is the `!important` necessary? (I know it was in the class, I just wonder if we could look at it case by case?)

Reply:
+ same for the two instances below